Source/Core/Common/EnumFormatter.h

```diff
@@ -60,22 +60,22 @@ class EnumFormatter
   template <typename FormatContext>
   auto format(const T& e, FormatContext& ctx)
   {
-    const auto value = static_cast<std::underlying_type_t<T>>(e);
+    const auto value_s = static_cast<std::underlying_type_t<T>>(e);
+    const auto value_u = static_cast<std::make_unsigned_t<T>>(value_s);
 
     if (!formatting_for_shader)
     {
-      if (value >= 0 && value < size && m_names[value] != nullptr)
-        return fmt::format_to(ctx.out(), "{} ({})", m_names[value], value);
+      if (value_s >= 0 && value_u < size && m_names[value_u] != nullptr)
```

Comment:
I'd suggest factoring this if expression into a bool above and using that bool here and on line 75.